src/middlewares/batchMiddleware.js

```diff
@@ -126,9 +126,10 @@ const queues = {};
  * @return {Middleware}
  * @warning This middleware do not propagate operation options
  */
-export default function(
-  batchedOperations = DEFAULT_BATCHED_OPERATIONS
-) {
+export default function({
+  batchedOperations = DEFAULT_BATCHED_OPERATIONS,
+  timeout = 0,
+} = {}) {
   return function batchMiddleware({controllerId, operationId}) {
     return next => function(params, callback, {batch = true} = {}) {
```

Comment:
- Comment arguments (like in lscache)
- Document middleware option "batch" in docs.